preset/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (api, options = { disableBabelPlugin: false }) {
+  if (options.disableBabelPlugin) {
+    return {};
+  }
+
+  const platform = api.caller(getPlatform);
+  const isWeb = platform === 'web';
+
+  return {
+    plugins: [['../plugin', { isWeb }]],
+  };
+};
+
+function getPlatform(caller) {
+  return caller && (caller.platform ?? caller.target);
+}
```

Comment:
I don't like the fact that `getPlatform` returns `platform` or `target`. How about renaming this function to `isWeb(caller)` and returning boolean from it?